scripts/configure-installer-tests-cluster.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+# Setup to find nessasary data from cluster setup
+## Constants
+HTPASSWD_FILE="./htpass"
+USERPASS="developer"
+HTPASSWD_SECRET="htpasswd-secret"
+# Overrideable information
+DEFAULT_INSTALLER_ASSETS_DIR=${DEFAULT_INSTALLER_ASSETS_DIR:-$(pwd)}
+KUBEADMIN_USER=${KUBEADMIN_USER:-"kubeadmin"}
+KUBEADMIN_PASSWORD_FILE=${KUBEADMIN_PASSWORD_FILE:-"${DEFAULT_INSTALLER_ASSETS_DIR}/auth/kubeadmin-password"}
+# Exported to current env
+export KUBECONFIG=${KUBECONFIG:-"${DEFAULT_INSTALLER_ASSETS_DIR}/auth/kubeconfig"}
+
+# List of users to create
+USERS="developer odonoprojectattemptscreateproject odosingleprojectattemptscreate odologinnoproject odologinsingleproject1"
+
+# Check if nessasary files exist
+if [ ! -f $KUBEADMIN_PASSWORD_FILE ]; then
+    echo "Could not find kubeadmin password file"
+    exit 1
+fi
+
+if [ ! -f $KUBECONFIG ]; then
+    echo "Could not find kubeadm password file"
+    exit 1
+fi
+
+# Get kubeadmin password from file
+KUBEADMIN_PASSWORD=`cat $KUBEADMIN_PASSWORD_FILE`
+
+# Remove existing htpasswd file, if any
+if [ -f $HTPASSWD_FILE ]; then
+    rm -rf $HTPASSWD_FILE
+fi
+
+# Set so first time -c parameter gets applied to htpasswd
+HTPASSWD_CREATED=" -c "
+
+# Create htpasswd entries for all listed users
+for i in `echo $USERS`; do
+    htpasswd -b $HTPASSWD_CREATED $HTPASSWD_FILE $i $USERPASS
+    HTPASSWD_CREATED=""
+done
+
+# Login as admin user
+oc login -u $KUBEADMIN_USER -p $KUBEADMIN_PASSWORD
+
+# Create secret in cluster, removing if it already exists
+oc get secret $HTPASSWD_SECRET -n openshift-config
```

Comment:
It is a little bit confusing to see an error like this in  it in the output
`Error from server (NotFound): secrets "htpasswd-secret" not found`

even if it is what we expect, it is not apparent from looking at logs

it might be better to redirect both stdout and stderr to /dev/null

Reply:
Updated